src/renderer/api/endpoints/deployment.api.ts

```diff
@@ -23,6 +25,25 @@ export class DeploymentApi extends KubeApi<Deployment> {
       }
     })
   }
+
+  restart(params: { namespace: string; name: string }) {
```

Comment:
JFYI:
https://github.com/kubernetes/kubectl/blob/release-1.16/pkg/polymorphichelpers/objectrestarter.go#L32 
I took this as a source of true